src/test/java/de/jetwick/snacktory/ArticleTextExtractorTest.java

```diff
@@ -2045,7 +2045,7 @@ public void testHeadlinesNews() throws Exception {
         assertEquals("http://www.headlines-news.com/2016/05/14/1202189/congress-has-a-constitutional-duty-to-fix-puerto-rico-debt-crisis", res.getCanonicalUrl());
         assertTrue(res.getText(), res.getText().startsWith("By Brian Robertson, contributor"));
         assertTrue(res.getText(), res.getText().endsWith("the mainland with their own serious Next »"));
-        compareDates("2016-05-14 17:02:26", res.getDate());
+        compareDates("2016-05-14 00:00:00", res.getDate());
```

Comment:
ArticleExtractor is using '14.05.2016' out of all the dates available in the article. So time and timezone doesn't really come into picture.